src/main/java/mods/eln/Eln.java

```diff
@@ -6639,9 +6912,31 @@ private void recipeMacerator() {
             new ItemStack[]{new ItemStack(Blocks.gravel)}, 1.0 * f));
         maceratorRecipes.addRecipe(new Recipe(new ItemStack(Blocks.gravel),
             new ItemStack[]{new ItemStack(Items.flint)}, 1.0 * f));
-
         maceratorRecipes.addRecipe(new Recipe(new ItemStack(Blocks.dirt),
             new ItemStack[]{new ItemStack(Blocks.sand)}, 1.0 * f));
+        double batf = 50000 * batteryCapacityFactor + f; //to prevent overunity
```

Comment:
How does it do so?

"Prevent overunity by using more power than a new battery stores."